src/components/productlistloadmore.jsx

```diff
@@ -0,0 +1,100 @@
+/**
+ * Copyright © 2018 Elastic Path Software Inc. All rights reserved.
+ *
+ * This is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This software is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this license. If not, see
+ *
+ *     https://www.gnu.org/licenses/
+ *
+ *
+ */
+import React from 'react';
+import { withRouter } from 'react-router';
+import PropTypes from 'prop-types';
+import intl from 'react-intl-universal';
+import { navigationLookup } from '../utils/CortexLookup';
+import { login } from '../utils/AuthService';
+
+import './productlistloadmore.less';
+
+class ProductListLoadMore extends React.Component {
+  static propTypes = {
+    dataProps: PropTypes.objectOf(PropTypes.any).isRequired,
+    handleDataChange: PropTypes.func.isRequired,
+  }
+
+  constructor(props) {
+    super(props);
+    const nextLink = props.dataProps.links.find(link => link.rel === 'next');
+    this.state = {
+      canLoadMore: Boolean(nextLink),
+      isLoading: false,
+    };
+
+    this.loadMore = this.loadMore.bind(this);
+  }
+
+  componentWillReceiveProps(nextProps) {
+    const nextLink = nextProps.dataProps.links.find(link => link.rel === 'next');
+    this.setState({ canLoadMore: Boolean(nextLink) });
+  }
+
+  loadMore() {
```

Comment:
Having some problems loading "more" products on the search result page (since that page shares this component). I'll keep checking and add any comments based on my findings:
![image](https://user-images.githubusercontent.com/39800563/51134988-fde4a000-1806-11e9-9307-d0731380267e.png)

Reply:
To reproduce:

• Enter a "space" key in the search bar
• search
• scroll down and click "load more"

Reply:
The loader from `ProductListItemMain` doesn't appear to get removed as the component gets updated.